_posts/2022-03-10-jump-1.0.md

```diff
@@ -0,0 +1,419 @@
+---
+layout: post
+title:  "JuMP 1.0.0 is released"
+date:   2022-03-10
+categories: [releases]
+permalink: /blog/1.0.0-release/
+---
+
+_Authors: Miles Lubin, Oscar Dowson, Iain Dunning, Joaquim Dias Garcia, Joey Huchette, and Benoît Legat._
+
+We are happy to announce the release of JuMP 1.0!
+
+Nearly 10 years in the making, the release of JuMP 1.0 represents a major
+milestone in the history of JuMP. It also represents a stable platform from
+which we can continue to build new and exciting features for a long time into
+the future.
+
+In this post we explain what "going 1.0" means for us now and what it will mean
```

Comment:
Somewhere in the post, we should link to the release notes and maybe summarize what's recently new with JuMP.